### PR TITLE
Modify print_init method

### PR DESCRIPTION
**Pull request by inseonglee**

### CODE REVIEW

**Comment by tikimtikim** [comment]
LGTM

**Comment by swmoon-unist** [comment]
lgtm

File: src/mqc/mqc.py [hunk 1]
```diff
@@ -311,12 +308,22 @@ def print_init(self, qm, mm, restart):
...
+
         # Print XF variables
         if (self.md_type == "SHXF"):
             # Print density threshold used in decoherence term
             dynamics_info += f"\n  Density Threshold        = {self.rho_threshold:>16.6f}"
+            # Print auxiliary trajectory setting
+            if (self.l_econs_state):
+                dynamics_info += f"\n  Aux. Total E             = {'Real Total E':>16s}"
+            else:
+                dynamics_info += f"\n  Aux. Velocity            = {'Real Velocity':>16s}"
```
Comment (jkha-unist) [inline src/mqc/mqc.py:324]: Depending on the option, the left hand side become different. I think this seems unclear and may cause confusion.

Comment (inseonglee) [reply src/mqc/mqc.py:324]: @tikimtikim @swmoon-unist There is one opinion that the confusion may exist in current init_message. Then, how to express 'l_econs_state' properly? Is there any better message?

Comment (tikimtikim) [reply src/mqc/mqc.py:324]: How about printing some sentences to explain how to determine velocity of aux. traj. when it is generated?
Ex.)
l_econs_state = {l_econs_state}
if (False):
  when auxiliary trajectory is generated, velocity of aux. traj. is same to velocity of real traj.
else:
  when auxiliary trajectory is generated, velocity of aux. traj. is rescaled to conserve total E of real traj.

Comment (jkha-unist) [reply src/mqc/mqc.py:324]: If true
  `Aux. Total E = Real Total E`
else
  `Aux. Total E = Real Total E + potential energy difference`?

Comment (tikimtikim) [reply src/mqc/mqc.py:324]: WFM

**Pull request merged**